docs/resources/obs_bucket_policy.md

```diff
@@ -0,0 +1,103 @@
+---
+subcategory: "Object Storage Service (OBS)"
+---
+
+# flexibleengine_obs_bucket_policy
+
+Attaches a policy to an OBS bucket resource.
+
+-> **NOTE:** When creating or updating the OBS bucket policy, the original policy will be overwritten.
+
+## Example Usage
+
+### Policy with OBS format
+
+```hcl
+resource "flexibleengine_obs_bucket" "bucket" {
+  bucket = "my-test-bucket"
+}
+
+resource "flexibleengine_obs_bucket_policy" "policy" {
+  bucket = flexibleengine_obs_bucket.bucket.id
+  policy = <<POLICY
+{
+  "Statement": [
+    {
+      "Sid": "AddPerm",
+      "Effect": "Allow",
+      "Principal": {"ID": "*"},
+      "Action": ["GetObject"],
+      "Resource": "my-test-bucket/*"
+    }
+  ]
+}
+POLICY
+}
+```
+
+### Policy with S3 format
+
+```hcl
+resource "flexibleengine_obs_bucket" "bucket" {
+  bucket = "my-test-bucket"
+}
+
+resource "flexibleengine_obs_bucket_policy" "s3_policy" {
+  bucket        = flexibleengine_obs_bucket.bucket.id
+  policy_format = "s3"
+  policy        = <<POLICY
+{
+  "Version": "2008-10-17",
+  "Id": "MYBUCKETPOLICY",
+  "Statement": [
+    {
+      "Sid": "IPAllow",
+      "Effect": "Allow",
+      "Principal": "*",
+      "Action": "s3:*",
+      "Resource": "arn:aws:s3:::my-test-bucket/*",
+      "Condition": {
+        "IpAddress": {"aws:SourceIp": "8.8.8.8/32"}
+      }
+    }
+  ]
+}
+POLICY
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) The region in which to create the OBS bucket policy resource. If omitted, the
+  provider-level region will be used. Changing this creates a new OBS bucket policy resource.
+
+* `bucket` - (Required, String, ForceNew) Specifies the name of the bucket to which to apply the policy.
```

Comment:
```md
* `bucket` - (Required, String, ForceNew) Specifies the name of the bucket to which to apply the policy.

  Changing this creates a new OBS bucket policy resource.
```
lost `Changin this ... resource`

Reply:
![image](https://github.com/FlexibleEngineCloud/terraform-provider-flexibleengine/assets/145660317/f5f2d82a-47b3-4bf1-b6d2-54e97b09e914)
revised